src/components/tag/tag.stories.js

```diff
@@ -1,13 +1,53 @@
 import { storiesOf } from '@storybook/vue'
+import Close from 'vue-material-design-icons/Close.vue'
 
 import aposTag from './tag.vue'
+// import aposIcon from '../icon/icon.vue'
 
 storiesOf('Tag', module)
   .add('Standalone', () => ({
     components: { aposTag },
     template: '<aposTag label="Portrait" />'
   }))
   .add('List', () => ({
-    components: { aposTag },
-    template: '<aposTag label="Portrait" />'
+    components: { aposTag, Close },
+    template: `
+      <ul class="apos-tag-list">
+        <li>
+          <aposTag label="Community">
+            <template v-slot:icon>
+              <Close title="Remove Tag" size="14" />
```

Comment:
For all these `size` props, need to use `:` to tell Vue it's a number and not a string, otherwise it fails the type check and gets mad in the console.
```
<Close title="Remove Tag" :size="14" />
```